content/en/apps/guides/database/muting_in_dashboards.md

```diff
@@ -0,0 +1,86 @@
+---
+title: "Contact Muting in SQL queries"
+linkTitle: "Contact Muting in SQL queries"
+weight: 
+description: >
+  How to write SQL queries excluding muted contacts correctly
+relevantLinks: >
+---
+
+When a contact gets muted, two of [many things]({{< ref "apps/reference/app-settings/transitions#muting" >}}) happen: 
+
+- The target contact and all of its descendants have a `muted` property set equal to the date they were muted
+- an entry is added to the contact's `muting_history` in sentinel's info docs
+
+When building dashboards on Superset, Klipfolio, or other data visualization platforms, you might need to exclude these muted contacts from the visualized data. An easy way to do this is to check the contact's `muted` property which when present has the date value of when the contact was muted and when absent means that the contact is not muted. This works when you are only interested in seeing the latest data but it gets complicated when you want to look at a contact's mute state from a certain period in the past.
+
+For example, if a contact was muted in February and unmuted in May; If you check the contact's mute state in March from June, you'd find that the contact would not have the muted property as it would have been removed during the `unmute` in May. This is where the `muting_history` comes in. A contact's `muting_history` contains all mute and unmute events stored in a JSON array. An example of the mute/unmute entries is shown below:
+
+```json
+{
+    "_id": "_id",
+    "_rev": "3-01ecfdd2958baeaf16fc621c5622f4a9",
+    "type": "info",
+    "doc_id": "doc-id",
+    "transitions": {
+    },
+    "muting_history": [
+        {
+            "date": "2021-04-07T13:41:09.769Z",
+            "muted": true,
+            "report_id": "_id"
+        },
+        {
+            "date": "2021-06-07T13:41:09.769Z",
+            "muted": false,
+            "report_id": "_id"
+        }
+    ],
+    "latest_replication_date": "2020-10-30T15:46:45.482Z",
+    "initial_replication_date": "2020-10-30T15:46:45.482Z"
+}
+```
+If you extract this data into a separate table you can get a timeline that you can use to check if the contact was muted at a certain point in the past. An example of this approach is shown below where the data is first extracted using the query:
+
+```sql
+    SELECT x.contact_uuid, date AS muted_on, muted, report_id FROM(
+        SELECT 
+            doc ->> 'doc_id' AS contact_uuid,
+            couchdb.doc ->> 'muting_history' AS muting_history
+        FROM 
+            couchdb
+        WHERE
+            doc ->> 'type' :: text = 'info'
+            AND doc ->> 'muting_history' IS NOT NULL
+    ) x
```

Comment:
For readability, could you change `x` to something more descriptive? Perhaps `contact_muting_history` or `muting_history`.